src/yuzu/main.cpp

```diff
@@ -1797,6 +1799,17 @@ void GMainWindow::OnMenuReportCompatibility() {
     }
 }
 
+void GMainWindow::OnSwitchModsPage() {
+    const std::string mods_page_url = "https://github.com/yuzu-emu/yuzu/wiki/Switch-Mods";
```

Comment:
```suggestion
    const auto mods_page_url = QStringLiteral("https://github.com/yuzu-emu/yuzu/wiki/Switch-Mods");
```

provides the same behavior without the need to convert between string types.